etc/network/interfaces

```diff
@@ -14,9 +14,17 @@ iface eth0 inet manual
  post-up /sbin/udhcpc -p /var/run/udhcpc-eth0.pid --script=/etc/udhcpc/kano.script -S -i eth0
  pre-down pkill -F /var/run/udhcpc-eth0.pid 2>/dev/null || /bin/true
 
+allow-hotplug eth1
+iface eth0 inet manual
+ post-up /sbin/udhcpc -p /var/run/udhcpc-eth0.pid --script=/etc/udhcpc/kano.script -S -i eth1
+ pre-down pkill -F /var/run/udhcpc-eth0.pid 2>/dev/null || /bin/true
```

Comment:
I think you were right @convolu , both eth lines should match the same interface?